src/soilbiogeochem/SoilBiogeochemCompetitionMod.F90

```diff
@@ -234,6 +237,9 @@ subroutine SoilBiogeochemCompetition (bounds, num_soilc, filter_soilc,num_soilp,
     real(r8) :: residual_smin_no3(bounds%begc:bounds%endc)
     real(r8) :: residual_plant_ndemand(bounds%begc:bounds%endc)
     real(r8) :: sminn_to_plant_new(bounds%begc:bounds%endc)
+!KO
+    real(r8) :: smin_nh4_vr_factor                                    ! factor to reduce smin_nh4_nr if use_fan is true
```

Comment:
seems like this isn't even being used, should all of the changes in this module be removed?